src/components/shared/hr/Hr.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import styles from './Hr.module.css';
+
+function Hr(): JSX.Element {
+    return <hr className={styles.hr} />;
```

Comment:
Meiner det ikkje er bra å bruke <hr> mtp. UU, men det er kanskje for det meste når det blir brukt i brødtekst? https://www.uutilsynet.no/wcag-standarden/tekst-og-struktur/226#:~:text=Unng%C3%A5%20ogs%C3%A5%20elementene%20%3Cspan%3E%2C%20%3Cbr%3E%20og%20%3Chr%3E%20for%20%C3%A5%20skape%20avsnitt.

Reply:
Hmm... Kan gjøre dette med CSS isteden. :smile: